### DISCUSSION TITLE
How to delete a ListView cell?

### DISCUSSION THREAD

**Discussion by mprogers** [dotnet/maui#10754] [Q&A]
How do I delete from a ListView? I'd like to be able to backswipe, or long press, but neither seem to be possible. It's hard to believe that such an important control doesn't have that capability, what am I missing?

**Comment by xtuzy** [comment]
you can add gesture to cell content, and remove this item in viewmodel

**Reply by rogersm-uwosh** [reply]
Thanks, xtuzy, I appreciate your response. Would you (or anyone else reading this) happen to know of any code that shows how to do this?
